client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/GetHelpPage.tsx

```diff
@@ -0,0 +1,238 @@
+
+  useEffect(() => {
+    if (listRef.current) {
+      listRef.current.scrollTop = listRef.current.scrollHeight;
+    }
+  }, [messages]);
+
+  return (
+    <List ref={listRef} className="get-help-page-list">
+      {messages.map((message, index) => (
+        <ListItem
+          key={index}
+          className={`get-help-page-listitem ${
+            message.sender === 'Codaveri'
+              ? 'message-codaveri'
+              : 'message-student'
+          }`}
+        >
+          <ListItemText
+            className={`get-help-page-listitemtext ${
+              message.sender === 'Codaveri'
+                ? 'message-codaveri'
+                : 'message-student'
+            }`}
+            primary={message.text}
+            secondary={
+              message.timestamp
+                ? moment(message.timestamp).format(SHORT_DATE_TIME_FORMAT)
+                : ''
+            }
+          />
+        </ListItem>
+      ))}
+    </List>
+  );
+};
+
+const InputArea: FC<{
+  loading: boolean;
+  suggestions: string[];
+  questionId: number;
+  answerId: number | undefined;
+  submissionId: string | null;
+  questionIndex: number;
+}> = ({
+  loading,
+  suggestions,
+  questionId,
+  answerId,
+  submissionId,
+  questionIndex,
+}) => {
+  const { t } = useTranslation();
+  const { formatMessage } = useIntl();
+  const dispatch = useAppDispatch();
+  const [input, setInput] = useState<string>('');
+
+  const handleSendMessage = async (message: string): Promise<void> => {
+    if (message.trim()) {
+      const successMessage = t(translations.liveFeedbackSuccess, {
+        questionIndex,
+      });
+      const noFeedbackMessage = t(translations.liveFeedbackNoneGenerated, {
+        questionIndex,
+      });
+      setInput('');
+      dispatch(generateUserRequest(questionId, answerId, message));
+      dispatch(
+        generateLiveFeedback({
+          submissionId,
+          answerId,
+          questionId,
+          successMessage,
+          noFeedbackMessage,
+        }),
+      );
+    }
+  };
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setInput(e.target.value);
+  };
+
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>): void => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      e.stopPropagation();
+      handleSendMessage(input);
+    }
+  };
+
+  return (
+    <Box className="get-help-page-box-column">
+      <Box className="get-help-page-box-center">
+        {suggestions.map((suggestion, index) => (
+          <Button
+            key={index}
+            disabled={loading}
+            onClick={() => handleSendMessage(suggestion)}
+            sx={{ mx: 1 }}
+            variant="outlined"
+          >
+            {suggestion}
+          </Button>
+        ))}
+      </Box>
+      <Box className="get-help-page-box-full-width">
+        <TextField
+          className="get-help-page-textfield"
```

Comment:
Unless there's a good justification, please use our existing styling (e.g. take the same one from the Programming qns generator) instead of creating new styles.

Reply:
I have converted to use the TextField in [common lib](https://github.com/Coursemology/coursemology2/blob/f8ea4cea53d878ff5379a6be52b384ccad5e0a18/client/app/lib/components/core/fields/TextField.tsx). In `Programming qns generator`,  they are using the `TableCell`.